<launch>
  <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen" respawn="true" respawn_delay="2">
  <param name="channel_type"        type="string" value="tcp"/>  
  <param name="tcp_ip"              type="string" value="192.168.0.7"/>  
  <param name="tcp_port"            type="int"    value="20108"/>
  <param name="frame_id"            type="string" value="laser"/>
  <param name="inverted"            type="bool"   value="false"/>
  <param name="angle_compensate"    type="bool"   value="true"/>
  </node>
</launch>
